set.mm

```diff
@@ -533959,6 +535030,74 @@ closed sets (see for example ~ zarcls0 for the definition of ` V ` ).
       LVTPDVQCVEVFVH $.
   $}
 
+  ${
+    zrhneg.1 $e |- L = ( ZRHom ` R ) $.
+    zrhneg.2 $e |- I = ( invg ` R ) $.
+    zrhneg.3 $e |- ( ph -> R e. Ring ) $.
+    zrhneg.4 $e |- ( ph -> N e. ZZ ) $.
+    $( Image of an inverse by the canonical homomorphism from the integers to a
```

Comment:
"Image" -> "Function value"? The term "image" is ususally used for sets ( `( f " A )`).